dev-libs/socketstream/socketstream-0.7.0-r2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="C++ Streaming sockets library"
+HOMEPAGE="http://socketstream.sourceforge.net/"
+SRC_URI="mirror://sourceforge/socketstream/${P}.tar.gz"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~sparc ~x86"
+IUSE="doc"
+
+DEPEND="doc? ( app-doc/doxygen )"
+
+DOCS="AUTHORS ChangeLog NEWS README* HACKING TODO"
```

Comment:
Please check if you can do without this line. At least `HACKING` and `TODO` are worthless. If necessary, make it an array and spell out the README* files: `README.{error_codes,io_flags,socket_options}`.

Reply:
Thanks for the info. Actually i've tried it without declaring it, but it wouldn't install the ```HACKING``` file. However since it's worthless i can simply remove it.
BTW, ```README.``` isn't needed since the default [1] already installs all ```README*``` files. Also ```TODO``` files are installed by default. Maybe pms should be updated if it's worthless?

[1]https://projects.gentoo.org/pms/latest/pms.html#x1-135003r4

Reply:
no, I just checked the content of this particular file, the comment was not meant in general. If all we lose is `HACKING`, then just drop the line.

Reply:
ahh, ok.. i understand. thanks for the clarification.